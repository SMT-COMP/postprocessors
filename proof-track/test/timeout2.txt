0.00/0.00	; run solver
607.90/600.03	/export/starexec/sandbox/solver/bin/starexec_run_default: line 6: 46299 Alarm clock             java -Xms1536m -Xmx50g -da -dsa -jar smtinterpol.jar -q -o continue-on-error=false "$BENCHMARK" > $TMPDIR/proof.txt 2> $TMPDIR/solver-error.log
607.90/600.03	; checking proof
607.90/600.03	EOF
