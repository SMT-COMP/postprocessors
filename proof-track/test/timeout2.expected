starexec-result=starexec-unknown
reason=error
